packages/back-end/types/scim.d.ts

```diff
@@ -0,0 +1,43 @@
+import { Request } from "express";
+import { OrganizationInterface } from "./organization";
+
+export type BaseScimRequest = Request & {
+  organization: OrganizationInterface;
+};
+
+export type ScimGetRequest = BaseScimRequest & {
+  params: {
+    id: string;
+  };
+};
+
+export type ScimPostRequest = BaseScimRequest & {
+  // TODO: The body might actually be encrypted
+  body: {
+    schemas: string[];
+    externalId: string;
+    displayName: string;
+    meta: {
+      resourceType: string;
+    };
```

Comment:
Can probably reference some sort of `SCIMRequest` type that includes all the properties we expect within a POST/UPDATE